List.css

```diff
@@ -0,0 +1,89 @@
+  .list-container 
+  {  
+    align-items: center;
+    background-color: var(--component-container-color);
+    border-radius: 0.3rem;
+    display: flex;
+    font-size: .9rem;
+    gap: 0.5rem;
+    justify-content: center;
+    margin: .5rem;
+    padding: 1rem;
+    width: 50rem;
+    height: auto;
+  }
+  .list-title {
+    padding: 1rem;
+    font-weight: 5px;
+    border-bottom: 1px solid black;
+    margin-bottom: 0.5rem;
+  }
+  .list {
+    background-color: var(--component-white-color);
+    padding: 1rem 0;
+    margin: 2.5rem;
+    width: 15.6rem;
+    max-width: 90%;
```

Comment:
According to me, the width should be 90% and the max-width should be 15.6rem. Try it out

Reply:
Done! Thank you for pointing it out.